docs/content/en/docs/reference/artifacts.md

```diff
@@ -489,6 +492,64 @@ These steps use `image-builder` to create an Ubuntu-based Amazon Machine Image (
    ```
    The exported image will be available at the location `s3://$AMI_EXPORT_BUCKET_NAME/$AMI_EXPORT_PREFIX/$EXPORT_IMAGE_TASK_ID.raw`.
 
+### Build Nutanix node images
+
+These steps use `image-builder` to create a Ubuntu-based image for Nutanix AHV and import it into the AOS Image Service.
+
+1. Download an [Ubuntu cloud image](https://cloud-images.ubuntu.com/releases/focal/release/ubuntu-20.04-server-cloudimg-amd64.img) for the build and upload it to the AOS Image Service using Prism. You will need to specify this image name as the `source_image_name` in the `nutanix-connection.json` config file specified below.
+
+1. Create a linux user for running image-builder.
+   ```bash
+   sudo adduser image-builder
+   ```
+   Follow the prompt to provide a password for the image-builder user.
+1. Add image-builder user to the sudo group and change user as image-builder providing in the password from previous step when prompted.
+   ```bash
+   sudo usermod -aG sudo image-builder
+   su image-builder
+   ```
+1. Install packages and prepare environment:
+   ```bash
+   sudo apt update -y
+   sudo apt install jq unzip make ansible -y
+   sudo snap install yq
+   echo "HostKeyAlgorithms +ssh-rsa" >> /home/$USER/.ssh/config
+   echo "PubkeyAcceptedKeyTypes +ssh-rsa" >> /home/$USER/.ssh/config
+   ```
+1. Get `image-builder`:
+    ```bash
+    cd /home/$USER
+    sudo wget https://anywhere-assets.eks.amazonaws.com/releases/bundles/19/artifacts/image-builder/0.1.2/image-builder-linux-amd64.tar.gz
+    sudo tar xvf image-builder*.tar.gz
+    sudo cp image-builder /usr/local/bin
+    ```
+1. Create a `nutanix-connection.json` config file. More details on values can be found in the [image-builder documentation](https://image-builder.sigs.k8s.io/capi/providers/nutanix.html). See example below:
+   ```json
+   {
+     "nutanix_cluster_name": "Name of PE Cluster",
+     "source_image_name": "Name of Source Image",
+     "image_name": "Name of Destination Image",
+     "nutanix_subnet_name": "Name of Subnet",
+     "nutanix_endpoint": "Prism Central IP / FQDN",
+     "nutanix_insecure": "false",
+     "nutanix_port": "9440",
+     "nutanix_username": "PrismCentral_Username",
+     "nutanix_password": "PrismCentral_Password"
+   }
+   ```
+
+1. Run `image-builder` with the following options:
+
+      * `--os`: `ubuntu`
+      * `--hypervisor`: For Nutanix use `nutanix`
+      * `--release-channel`: Supported EKS Distro releases include 1-20, 1-21, 1-22, and 1-23.
+      * `--nutanix-config`: Nutanix configuration file (`nutanix-connection.json` in this example)
+
+      ```bash
+      cd /home/$USER
+      image-builder build --os ubuntu --hypervisor nutanix --nutanix-config nutanix-connection.json --release-channel 1-23
```

Comment:
let's mention `1-24` since that's the default k8s version now
```suggestion
      image-builder build --os ubuntu --hypervisor nutanix --nutanix-config nutanix-connection.json --release-channel 1-24
```